dagger/dag_creator/airflow/dag_creator.py

```diff
@@ -48,38 +49,44 @@ def execution_date_fn(execution_date, **kwargs):
 
         return execution_date_fn
 
-    def _get_external_task_sensor(self, from_task_id: str, to_task_id: str) -> ExternalTaskSensor:
+    def _get_external_task_sensor_name(self, from_task_id: str):
         from_pipeline_name = self._task_graph.get_node(from_task_id).obj.pipeline_name
         from_task_name = self._task_graph.get_node(from_task_id).obj.name
+        return f"{from_pipeline_name}-{from_task_name}-sensor"
+
+    def _get_external_task_sensor(self, from_task_id: str, to_task_id: str) -> ExternalTaskSensor:
```

Comment:
Can we add docstring?

Reply:
the docstring was not implemented in every function here. So Im wondering whether I should add the docstring here 🙈

Reply:
@vlad-choco Added the docstring :)